config/initializers/gds_api.rb

```diff
@@ -2,7 +2,13 @@
 require "gds_api/asset_manager"
 require "plek"
 
-GdsApi::Base.logger = Logger.new(Rails.root.join("log/#{Rails.env}.api_client.log"))
+GdsApi::Base.default_options = {
+  logger: if ENV["RAILS_LOG_TO_STDOUT"].present?
+            Logger.new($stdout)
+          else
+            Logger.new(Rails.root.join("log/#{Rails.env}.api_client.log"))
+          end,
+}
```

Comment:
I've no idea why this application sets up a dedicated log file for GDS API Adapters, it looks very inconsitent with the rest of the stack. I'd suggest deleting this config.

```suggestion
```

Reply:
Ooh cool, thanks! There are a couple of others which do the same thing ([smart-answers and service-manual-publisher](https://sourcegraph.com/search?q=context:global+repo:alphagov+api_client.log&patternType=literal)). Happy to ditch it from those too if you are.

Reply:
Ah nice spot - that sourcegraph website looks really helpful!

Yes, they can go too. Feel free to tag me on reviews since I have the context.